xbmc/music/ContextMenus.cpp

```diff
@@ -35,6 +36,22 @@
 namespace CONTEXTMENU
 {
 
+static bool IsMusicDbItem(const CFileItem& item)
+{
+  if (URIUtils::IsMusicDb(item.GetPath()))
```

Comment:
This if needs to go the `MUSICDATABASEDIRECTORY` will take care of it even if it's more expensive